staging/src/k8s.io/api/core/v1/types.go

```diff
@@ -7388,6 +7418,8 @@ const (
 	ResourceLimitsEphemeralStorage ResourceName = "limits.ephemeral-storage"
 	// resource.k8s.io devices requested with a certain DeviceClass, number
 	ResourceClaimsPerClass string = ".deviceclass.resource.k8s.io/devices"
+	// Resource device class prefix
+	ResourceDeviceClassPrefix string = "deviceclass.resource.kubernetes.io/"
```

Comment:
not a big deal, but at first glance, this *probably* belongs in the resource types.go file